EpsilonDriverControls/Src/DriverControls.c

```diff
@@ -140,29 +140,43 @@ void sendDriveCommandsTask(void const* arg)
     uint8_t prevResetStatus = 0;
     float dataToSendFloat[2];
 
+    uint8_t regenQueueIndex = 0;
+    uint8_t accelQueueIndex = 0;
+
     for (;;)
     {
         osDelayUntil(&prevWakeTime, DRIVE_COMMANDS_FREQ);
 
         // Read analog inputs
         if (HAL_ADC_PollForConversion(&hadc1, ADC_POLL_TIMEOUT) == HAL_OK)
         {
-            regenPercentage = ((float)HAL_ADC_GetValue(&hadc1)) / ((float)MAX_ANALOG);
+            newRegen = ((float)HAL_ADC_GetValue(&hadc1)) / ((float)MAX_ANALOG);
```

Comment:
This will always be zero, as the float will be converted to an int later, this will apply to other similar calculations.